### PR TITLE
21560 - EFT Payment reminder template updates

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:*
https://github.com/bcgov/entity/issues/21560

*Description of changes:*
- Update EFT payment reminder / due templates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: queue_services/account-mailer/src/account_mailer/email_templates/payment_due_notification.html [hunk 1]
```diff
@@ -1,12 +1,18 @@
-# Payment Due Today
+# Payment Due for {{ account_number }}: {{ account_name_with_branch }}
 
-## {{ account_number }}: {{ account_name_with_branch }}
+{% if short_name_links_count > 1 %}
+    There's an amount owing on your account for your {{ statement_month }} statement (#{{ statement_number }}), which is due today {{ due_date }}.
+{% else %}
+    There's an amount owing of {{ total_amount_owing }} on your account for your {{ statement_month }} statement (#{{ statement_number }}), which is due today {{ due_date }}.
```
Comment (seeker25) [inline queue_services/account-mailer/src/account_mailer/email_templates/payment_due_notification.html:6]: Isn't it possible to combine these a bit more?

Comment (seeker25) [reply queue_services/account-mailer/src/account_mailer/email_templates/payment_due_notification.html:6]: GPT:

```
{% set owing_message = "There's an amount owing" %}
{% set amount_details = '' if short_name_links_count > 1 else ' of {{ total_amount_owing }}' %}
{{ owing_message }}{{ amount_details }} on your account for your {{ statement_month }} statement (#{{ statement_number }}), which is due today {{ due_date }}.
```

Comment (ochiu) [reply queue_services/account-mailer/src/account_mailer/email_templates/payment_due_notification.html:6]: Possibly, I prefer the whole sentence rather trying to combine fragments though when we have to modify it.

File: queue_services/account-mailer/src/account_mailer/email_templates/payment_reminder_notification.html [hunk 2]
```diff
@@ -1,12 +1,18 @@
-# Payment Reminder
+# Payment Reminder for {{ account_number }}: {{ account_name_with_branch }}
 
-## {{ account_number }}: {{ account_name_with_branch }}
+{% if short_name_links_count > 1 %}
```
Comment (seeker25) [inline queue_services/account-mailer/src/account_mailer/email_templates/payment_reminder_notification.html:3]: same as above

**Review by seeker25 — APPROVED** [review]

File: queue_services/account-mailer/src/account_mailer/email_templates/payment_due_notification.html (hunk 1 above)
Comment (seeker25) [reply queue_services/account-mailer/src/account_mailer/email_templates/payment_due_notification.html:6]: ok, then you'll be modifying it twice if the base sentence needs to change

**Pull request merged**